spec/project-test-helpers.js

```diff
@@ -0,0 +1,90 @@
+/*!
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+*/
+
+const path = require('path');
+const fs = require('fs-extra');
+const { ConfigParser } = require('cordova-common');
+
+const fixturesPath = path.join(__dirname, 'cordova/fixtures');
```

Comment:
```suggestion
const fixturesPath = path.join(__dirname, 'cordova', 'fixtures');
```

Reply:
Actually, I usually change the for you suggested to the one I used. I like it better because I find it more legible and it is shorter. The result is the same in either case:
```js
require('path').win32.join('foo', 'bar/baz')
// => 'foo\\bar\\baz'
```